test

```diff
@@ -191,9 +191,8 @@ function integration_pass {
 		if [ -z "${USERPKG}" ]; then
 			INTEGTESTPKG=("${REPO_PATH}/integration"
 						  "${REPO_PATH}/client/integration"
-						  "${REPO_PATH}/clientv3/integration"
-						  "${REPO_PATH}/contrib/raftexample"
-						  "${REPO_PATH}/store")
```

Comment:
Since we don't have store package, it's safe to remove this.

Reply:
Good catch. Since it isn't introduced in this PR, feel free to do it in a separate PR.

Reply:
Failed to test each individual test case if this package is included. 

For example 

```
dev-dsk-chaochn-2c-a26acd76 % PASSES=integration TESTCASE="\bTestEtcdGrpcResolver\b" TIMEOUT=1m ./test
Running with TEST_CPUS: 1,2,4
Starting 'integration' pass at Fri Oct 20 15:44:26 UTC 2023
Running integration tests...
go: downloading google.golang.org/grpc v1.26.0
no required module provides package go.etcd.io/etcd/store; to add it:
	go get go.etcd.io/etcd/store
```